README.md

```diff
@@ -1073,6 +1073,21 @@ The following script parameters are available:
   being able to be active again. Useful for preventing constant
   teleportation.
 
+##### VRTK_RoomExtender
```

Comment:
I'd move this documentation to be underneath the section:
`Touchpad Movement (VRTK_TouchpadWalking)`

As it's all grouped with locomotion stuff